runtime/include/j9cfg.h.in

```diff
@@ -171,6 +171,7 @@ extern "C" {
 #cmakedefine J9VM_INTERP_ROMABLE_AOT_SUPPORT
 #cmakedefine J9VM_INTERP_SIG_QUIT_THREAD
 #cmakedefine J9VM_INTERP_SIG_QUIT_THREAD_USES_SEMAPHORES
+#cmakedefine J9VM_INTERP_SIG_USR2
```

Comment:
Equivalent changes for UMA builds are missing, e.g. `f9.flags`, `buildspecs/*.spec`, etc.

Reply:
Also, `se6_vm-side_natives_objects.xml` needs to (conditionally) include `sigusr2`:
```
	<object name="sigusr2">
		<include-if condition="spec.flags.interp_sigUsr2" />
	</object>
```